### DISCUSSION TITLE
Resource crunch with multiple services being up crashes few services when using colima.

### DISCUSSION THREAD

**Discussion by linkwithsaurav** [abiosoft/colima#645] [Q&A]
I have nearly 31 services attached in my docker-compose, and using ```colima start --cpu 8 --memory 12 --disk 120```  on a Mac M1 chip with 16 GB RAM but some services keep exiting after few minutes, we have found that this occurs due to resource crunch, is there any solution that someone is using with micro-service testing on local, tried out a lot of configuration of colima but the result was same.

**Comment by abiosoft** [comment]
Were you able to verify the logs of the exiting services and ascertain that it is indeed due to resource crunch?

By resource crunch, are you referring to CPU or memory or both?

Also, you might have better success allocating less resources to the VM i.e. leaving enough for the host. On a cpu with 8 cores and 16gb ram, I would recommend allocating 4 cores and 8gb ram to the VM.

**Reply by linkwithsaurav** [reply]
I have tried with a lot of different configurations- the extreme config that I allocated was memory about 14 gb and core 8 and disk space about 120gb, I see intel mac able to run the same services with ease but new arm macs show issue with colima
